Source/Renderer/Texture.js

```diff
@@ -281,13 +283,13 @@ define([
                         magFilter = TextureMagnificationFilter.NEAREST;
                     }
 
-                    sampler = {
+                    sampler = new Sampler({
```

Comment:
I had considered that, but decided to stick to what was already there. I can change it.